package/MDAnalysis/core/universe.py

```diff
@@ -312,11 +317,14 @@ class Universe(object):
     .. versionchanged:: 2.0.0
         Universe now can be (un)pickled.
         ``topology`` and ``trajectory`` are reserved upon unpickle.
-
+    .. versionchanged:: 2.5.0
+        Added fudge_factor and lower_bound parameters for use with
+        **guess_bonds**.
```

Comment:
Single asterisk to be consistent with rest of docs.
 
```suggestion
        *guess_bonds*.
```